taskvine/src/bindings/python3/taskvine.binding.py

```diff
@@ -116,20 +132,19 @@ def __init__(self, url):
 #
 # A file obtained from a buffer in memory.
 
-
 class FileBuffer(File):
     ##
     # Create a file from a buffer in memory.
     #
     # @param self       The current file object.
-    # @param name       The abstract name of the buffer.
-    # @param buffer     The contents of the buffer.
-
-    def __init__(self, name, buffer):
-        name = str(name)
-        buffer = str(buffer)
-        self._file = vine_file_buffer(name, buffer, len(buffer))
+    # @param buffer     The contents of the buffer, or None for an empty output buffer.
 
+    def __init__(self, buffer=None):
+        if buffer is None:
+            self._file = vine_file_buffer(None,0);
+        else:
+            buffer = str(buffer)
```

Comment:
I don't think you want str, as that calls a method on buffer which may not give you what you want. Suggest for now:
```
buffer = bytes(buffer, "ascii")
```

In the future we may want to use memory views:  https://stackoverflow.com/questions/16998686/expose-a-vector-as-a-memoryview-using-swig